### ISSUE TITLE
HaplotypeCaller core dump on selected jobs

### ISSUE THREAD

**Issue by wenluo711** [broadinstitute/gatk#7515]
Hi GATK developers:

Have 4 pacbio WGS bam files do to Haplotype calling. Each bam file was divided by chromosomes, but 3 parallele jobs failed due to java core dump:
- Syntax I ran was pretty basic, I also tried latest gatk version4.2.2.0, same result. Java version is ``` OpenJDK Runtime Environment (build 1.8.0_252-b09) ```
```
/gatk-4.0.11.0/gatk --java-options "-Xmx4G" HaplotypeCaller  \
           -R GRCh38.p2.fa  \
           -I RT4_STD.bam  \
           -ERC GVCF  \
           -L chr16  \
           -O RT4_STD.g.vcf  \
           -new-qual
```
- Error message is also different
- First one is :
```
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00002aaad9f1e54a, pid=7818, tid=0x00002aaaabdce700
#
# JRE version: Java(TM) SE Runtime Environment (8.0_111-b14) (build 1.8.0_111-b14)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.111-b14 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  [libgkl_pairhmm_omp1890484777463615571.so+0x6954a]  double compute_full_prob_avxd<double>(testcase*)+0x34a
#
# Core dump written. Default location: core or core.7818
#
# An error report file with more information is saved as:
# hs_err_pid7818.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
```

-Second one is 
```
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00000035dfe84364, pid=160107, tid=0x00002aaaabdce700
#
# JRE version: Java(TM) SE Runtime Environment (8.0_111-b14) (build 1.8.0_111-b14)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.111-b14 mixed mode linux-amd64 )
# Problematic frame:
# C  [libc.so.6+0x84364]
#
# Core dump written. Default location: core or core.160107
#
# An error report file with more information is saved as:
# hs_err_pid160107.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.

```

Could this be a bug of problem with my data?

Thanks, 
Wen

**Comment by wenluo711** [comment]
resolved with version 4.2.2

**Comment by droazen** [comment]
@wenluo711 Glad to hear this was resolved in 4.2.2! The newer releases of GATK contain an updated GKL library that fixes many of these segmentation fault / core dump related crashes.
